modules/phase_field/include/postprocessors/FeatureFloodCount.h

```diff
@@ -91,18 +91,13 @@ class FeatureFloodCount :
     {
     }
 
-    FeatureData(const FeatureData & f) :
-        _ghosted_ids(f._ghosted_ids),
-        _local_ids(f._local_ids),
-        _halo_ids(f._halo_ids),
-        _periodic_nodes(f._periodic_nodes),
-        _var_idx(f._var_idx),
-        _bboxes(f._bboxes),
-        _min_entity_id(f._min_entity_id),
-        _status(NOT_MARKED),
```

Comment:
Wow - good catch! I'd better make sure that this isn't a problem before this is merged.